examples/non-fungible/src/contract.rs

```diff
@@ -260,27 +263,28 @@ impl NonFungibleToken {
         }
     }
 
-    async fn get_nft(&self, token_id: &TokenId) -> Nft {
-        self.nfts
+    async fn get_nft(&self, token_id: &TokenId) -> Result<Nft, Error> {
+        Ok(self
+            .nfts
             .get(token_id)
-            .await
-            .expect("Failure in retrieving NFT")
-            .expect("NFT should not be None")
+            .await?
+            .expect("NFT should not be None"))
```

Comment:
Isn't it the other way round? We can expect to not get a `ViewError`, but the NFT could be `None`?

Reply:
Right, then I can return an error if the NFT is `None`, right? Since I always expect it to exist when this is called